listenbrainz/spark/request_manage.py

```diff
@@ -133,6 +133,27 @@ def request_sitewide_stats(range_, entity):
         click.echo("Incorrect arguments provided")
 
 
+@cli.command(name="request_new_release_stats")
```

Comment:
Should this maybe just be one entry point to calculate all year end stats?

Reply:
Yes, will add a single entry point as well. For now, separate ones make sense. Easier to test changes in single stat.